stub_rule.go

```diff
@@ -209,3 +209,14 @@ func (s *StubRule) MarshalJSON() ([]byte, error) {
 
 	return json.Marshal(jsonStubRule)
 }
+
+//MarshalJSON makes json body for request to find requests
+//adding a separate MarshalJSON method for the request object
+//as it is required to convert the request to JSON
+func (r *request) MarshalJSON() ([]byte, error) {
+	req := map[string]interface{}{
```

Comment:
It would be great if we could use the maximum number of the request [fields].(http://wiremock.org/docs/api/#tag/Requests/paths/~1__admin~1requests~1find/post)
Can we move [all parsing of the request](https://github.com/walkerus/go-wiremock/blob/main/stub_rule.go#L169-L208) here and use the original structure [here](https://github.com/walkerus/go-wiremock/blob/main/stub_rule.go#L155)?

Reply:
makes complete sense will address this